ferret/benchmark.py

```diff
@@ -47,10 +47,14 @@ def lp_normalize(explanations, ord=1):
 
     new_exps = list()
     for exp in explanations:
+        if isinstance(exp, tuple): # some explainers might return a tuple to handle the multiple-choice task.
```

Comment:
If not a find of big if-switches -- can't we make this a property of the `Explanation` class and let the benchmark code access it? Are there better options?

Reply:
Yes, you are right. This was introduced to handle the Multiple Choice Task, which is not related to the present PR. I'll simply remove the check, as it is not needed here and might create some confusion.